app/src/components/market/common/arbitrators/index.tsx

```diff
@@ -14,12 +14,15 @@ const Wrapper = styled.div`
 `
 
 const ArbitratorButton = styled(Button)<{ isSelected: boolean }>`
-  margin: 0 8px 12px 0;
+  margin: 0 12px 12px 0;
   &,
   &:hover {
     border-color: ${props =>
       props.isSelected ? props.theme.textfield.borderColorActive : props.theme.buttonSecondaryLine.borderColor};
   }
+  &:hover {
+    border-color: ${props => props.theme.textfield.borderColorActive};
+  }
```

Comment:
Good job! This works i would just add small optimisation...I played around with it and seems that if you just add borderColorActive to props.isSelected else statment it will work just the same? maybe i'm missing something...

Reply:
this doesn't work for me

Reply:
i cleaned up the code a bit though check out new changes?